apps/src/storage/dataBrowser/LibraryTable.jsx

```diff
@@ -74,16 +74,16 @@ class LibraryTable extends React.Component {
 
   importTable = datasetInfo => {
     if (datasetInfo.current) {
-      // TODO: Implement current tables (see STAR-615)
+      FirebaseStorage.addCurrentTableToProject(
+        datasetInfo.name,
+        () => console.log('success'),
```

Comment:
Is there still a plan to replace these console logs?

Reply:
https://codedotorg.atlassian.net/browse/STAR-672 covers adding warning messages, and we can remove these console logs at that point